shuriken/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ project(
         HOMEPAGE_URL https://github.com/Fare9/Shuriken-Analyzer/
 )
 
+add_definitions(-DSHURIKENLIB_EXPORTS)
```

Comment:
Does this flag need to be included somewhere in the documentation for compiling Shuriken? Just in case to add it to the README.md or somewhere else.

Reply:
its needed by api\C\shuriken_core.h and other places because of:

```
#if defined(_WIN32) || defined(__WIN32__)
#ifdef SHURIKENLIB_EXPORTS
#define SHURIKENCOREAPI __declspec(dllexport)
#else
#define SHURIKENCOREAPI __declspec(dllimport)
#endif
```

so when shuriken_lib builds it will turn on the exporting of the API etc. i think it should stay inside this cmake file so that this process is transparent.

normally when client code uses the api header then this flag defaults to import shuriken lib, so i dont think its required in any documentation

Reply:
okay, okay, that you are directly adding into the CMake, okay, it is solved for me then.